cookbook/basic_rag_example/local_embeddings.py

```diff
@@ -0,0 +1,60 @@
+"""A simple script for using RAG to ask questions about recent soccer results."""
+import os
+
+import numpy as np
+import openai
+import pandas as pd
+
+from mirascope import OpenAIChat, Prompt
+
+os.environ["OPENAI_API_KEY"] = "YOUR_API_KEY"
+
+soccer_info = [
+    "Manchester City won the league, continuing their domestic dominance.",
+    "In La Liga, FC Barcelona emerged victorious with a sturdy defense.",
+    "Dortmund slipped up in the last matchday to hand Bayern the Bundesliga title.",
+    "For the first time since Diego Maradona's days, Napoli lifted the Serie A trophy.",
+]
+
+client = openai.OpenAI()
```

Comment:
any reason why you are using openai instead of mirascope OpenAIChat?

Reply:
for the embeddings. see line 24

Reply:
but we can use embeddings in our wrapper, is the idea that we wont use our wrapper for anything other than our prompts?

Reply:
On second thought yes, I think it probably makes the most sense to access embeddings through Mirascope since this is a Mirascope cookbook not an OpenAI tutorial